Assets/TextAssets/TXT/MemoriesTest.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+/* Author: Anisha Pai
+ * Description: Tests converting a single json file
+ */
+
+public class Memories : MonoBehaviour 

```

Comment:
Nice rename! Just want to make sure the class name is now also `MemoriesTest` to match